contrib/pg_tde/pg_tde--1.0-rc.sql

```diff
@@ -41,13 +41,13 @@ BEGIN ATOMIC
     -- load_vaultV2_keyring_provider_options function.
     SELECT pg_tde_add_database_key_provider('vault-v2', provider_name,
                             json_object('url' VALUE COALESCE(vault_url, ''),
-                            'token' VALUE COALESCE(vault_token, ''),
+                            'tokenPath' VALUE COALESCE(vault_token_path, ''),
                             'mountPath' VALUE COALESCE(vault_mount_path, ''),
                             'caPath' VALUE COALESCE(vault_ca_path, '')));
 END;
 
 CREATE FUNCTION pg_tde_add_database_key_provider_vault_v2(provider_name TEXT,
-                                                vault_token JSON,
+                                                vault_token_path JSON,
```

Comment:
Do we even need to JSON functions anymore?

Reply:
No, we should be able to remove these completely.